cpp/modmesh/view/RManager.cpp

```diff
@@ -212,19 +212,31 @@ void RManager::setUpMenu()
 
     m_oneMenu = m_mainWindow->menuBar()->addMenu(QString("One"));
     m_oneMenu->addAction(new RAppAction(
-        QString("Load euler1d"),
-        QString("Load euler1d"),
+        QString("Euler solver"),
```

Comment:
Piggyback on this PR to rename the Euler solver menu item.  It (should) still use the "app".